application-ckeditor-ui/src/main/resources/CKEditor/VelocityMacros.xml

```diff
@@ -214,12 +214,14 @@
   #end
   #set ($officeImporterSupported = $officeServerConnected &amp;&amp;
     "$!services.wysiwyg.importOfficeAttachment($missingAttachmentReference, {'useOfficeViewer': true})" != '')
+  #set ($temporaryAttachmentSupported = "$!services.temporaryAttachments" != '')
```

Comment:
I don't think this is needed anymore now that the config is done in ``ConfigSheet``.